paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsViewModel.kt

```diff
@@ -329,17 +330,22 @@ internal class PaymentOptionsViewModel @Inject constructor(
         val target = if (args.state.showSavedPaymentMethods) {
             SelectSavedPaymentMethods(getCvcRecollectionState())
         } else {
-            AddFirstPaymentMethod
+            AddFirstPaymentMethod(interactor = DefaultAddPaymentMethodInteractor(this))
         }
 
+        val sheetViewModel = this
```

Comment:
Don't need to change, but there's probably another way to do the same thing.

`this@PaymentOptionsViewModel` instead of `sheetViewModel` down below.